### ISSUE TITLE
convert_model.py fails when a custom loss function was used in training

### ISSUE THREAD

**Issue by dasmysh** [Dobiasd/frugally-deep#397]
When loading a Keras model with `load_model(in_parth)` (see https://github.com/Dobiasd/frugally-deep/blob/5784f91382313b1d1d95fa7c08262f9a8c506fd0/keras_export/convert_model.py#L871)
the model will be compiled automatically. That fails when a custom loss function was used in training before.
Since you compile the model later anyways, you could skip the compilation in `load_model` by using `compile=False` as a parameter:
`load_model(in_path, compile=False)`. This fixed the issue for me while the conversion still worked fine.

**Comment by Dobiasd** [comment]
Oh, thanks a lot for the remark! :+1:

You're absolutely right. So I just pushed a commit with your suggested change. :heavy_check_mark:
